meta-sokol-flex-staging/security/recipes-ids/tripwire/tripwire_%.bbappend

```diff
@@ -3,7 +3,7 @@
 # ---------------------------------------------------------------------------------------------------------------------
 
 # Use vi by default to avoid pulling in nano
-RDEPENDS_REMOVE = "nano"
+RDEPENDS_REMOVE:feature-sokol-flex-staging = "nano"
 RDEPENDS:${PN}:remove:feature-sokol-flex-staging = " ${RDEPENDS_REMOVE}"
```

Comment:
This isn't needed, we already use this override here, right below that line, where we actually use this variable.

Reply:
Must have been doing this late at night. * smh *